docs/api.json

```diff
@@ -4382,7 +4382,7 @@
         }
       },
       "level2Ability": {
-        "description": "Hurt: Gain Coconut Shield.",
+        "description": "Hurt: Gain Coconut Shield. Works 2 times per turn.",
```

Comment:
Hi @danieldides 

Thanks for the contribution!

Would it be possible to make these changes to `src/db/pets/gorilla.ts` and then run `npm run prod:build`? Everything on the site is generated from those files (including this `docs/api.json` file) - so updating there should ripple your changes out to all the pages.

Reply:
Doh! Took another stab at this. Never really worked with a system like this before so I hope I did it right this time. Let me know how it looks.